data_collection/other_integrations/web_log_analytics.rst

```diff
@@ -101,3 +105,29 @@ Supported log formats:
 - log files of some popular Cloud Saas services: Amazon CloudFront logs, Amazon S3 logs
 - streaming media server log files such as: Icecast
 - log files with and without the virtual host will be imported
+
+The script does not directly support importing logs from log aggregator tools, like Grafana Loki or ELK. If you'd like to import logs from one of those, you need to download them to the disk first.
+
+
+Performance considerations & rate limiting
+----------------------
+
+The script needs CPU to read and parse the log files, but it is usually Piwik PRO server itself which will limit the import speed due to network latency.
+To improve performance, you can use the **--recorders** option to specify the number of parallel threads which will import hits into Piwik PRO. We recommend to increase **--recorders=N** until import speed is satisfying for you.
+
+If you are Piwik PRO Core user, please make sure, that you are not hitting rate limits, by using **--debug-force-one-hit-every-Ns** flag to slow down the import process.
```

Comment:
I may be wrong here but the payload is not restricted by the rate limiter.
If so then keeping the `--recorders` low and `--recorder-max-payload-size` high will grant you a great throughput while keeping the actual requests to the tracker relatively low.
And this is the scenario we recommend.

Reply:
Damn... I just realized that `--enable-bulk-tracking` is OFF by default (it was ON for Rancher version). @sebastianpiskorski  / @RomanSek do you remember why?

My general recommendation for any log import would be:
* --recorders=2
* --max-payload-size=1000
* --enable-bulk-tracking=1
Those are IMO the best values to start with for Promil setup.

Reply:
I think that we disabled `--enable-bulk-tracking`, because we mostly did so on Rancher, since it killed instances instantly. With new tracker, I think you have the point that we can provide default values tweaked already. So it would be:

--enable-bulk-tracking=1 - since it would be faster and new tracker should handle it eaisily
--recorders=4 - since on infras with HA this is CollectorFront max replicas value
--max-payload-size=95 - since this is max that can be send with each request maximized (POST limit payload is 6MiB and each request in bulk can have 64kiB, 6144÷64=96), minus 1 for additional JSON characters, like wrapping array, commas and so on.

Reply:
As for limits on core it is 70req/s (it think that there no distinction between GET and POST), so `--debug-force-one-hit-every-Ns` would useful if we would change it to miliseconds, instead of seconds.

Reply:
To summarize, I changed the defaults in the script:
* we are using bulk tracking by default
* we are using max-payload-size=95
* we are using one recorder

Also I changed `--debug-force-one-hit-every-Ns` to `--sleep-between-requests-ms`